wallet-new/integration/Util.hs

```diff
@@ -170,6 +171,22 @@ noLongerThan action maxWaitingTime = do
         Left _  -> return Nothing
         Right a -> return $ Just a
 
+confirmTransaction
+    :: [Transaction]
+    -> Transaction
+    -> PropertyM IO ()
+confirmTransaction transactions txn = do
+    let txnEntry: _ = filter ( \x -> (txId x) == (txId txn)) transactions
+    log $ "Resp   : " <> ppShowT txnEntry
+    liftIO $ [txStatus txnEntry] `shouldContain` [InNewestBlocks, Persisted]
```

Comment:
I read it the other way around. Isn't `[InNewestBlocks, Persisted]` which should contain `[txStatus txnEntry]` ?